Garina tasks  second part.js

```diff
@@ -0,0 +1,27 @@
+// Extracts a name from template string 'Hello, First_Name Last_Name!'
+
+function extractNameFromTemplate(value) {
+    let c = value.replace("Hello,", '')
+    let d = c.replace("!", '')
+    return d
+    }
```

Comment:
Good job.
But you could try to re-write in one line(action) and it will look more cool

Reply:
`function extractNameFromTemplate(value) { 
return value.replace("Hello,", '').replace("!", '') 
}`